### PR TITLE
Feature/build swagger on ci

### LINKED ISSUE #1770: switch off multi-buildpack
Make sure docs still work.

### PR DESCRIPTION
**Pull request by adborden**
This get's us off of the multibuildpack which is deprecated. Thanks to @jmcarp for getting most of it done. The last piece is getting swagger-ui out of node_modules so we're not shipping any dev dependencies to cloud.gov. This reverts the last fix which saved us from a broken swagger docs page and adds a quick build step to pull the built swagger-ui assets out of `node_modules`.

I tested this via [travis](https://travis-ci.org/18F/openFEC/builds/143403613) on the feature space, so hopefully there's no more surprises.

Fixes #1770

### CODE REVIEW

**Comment by ccostino** [comment]
Thank you, @adborden!  I'll take a closer look for review - does this still require we install `swagger-tools` for ourselves locally per the current README instructions (`npm install -g swagger-tools`), or is it possible we can place that in the `package.json` requirements as well to keep dependencies local to the app?

File: package.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
   "dependencies": {
      "swagger-ui": "git+https://github.com/18F/swagger-ui.git"
   },
+  "scripts": {
+    "build": "mkdir -p static/swagger-ui && cp -R node_modules/swagger-ui/dist static/swagger-ui/"
+  },
   "engines": {
     "node": "0.12.7",
```
Comment (ccostino) [inline package.json]: We should really update this...  Separate PR though to test.

**Comment by ccostino** [comment]
Thanks again, @adborden, we're going to merge this and can address additional improvements in a subsequent PR(s) if need be!

**Comment by adborden** [comment]
> Thank you, @adborden! I'll take a closer look for review - does this still require we install swagger-tools for ourselves locally per the current README instructions (npm install -g swagger-tools), or is it possible we can place that in the package.json requirements as well to keep dependencies local to the app?

From slack conversations `swagger-tools` might still be required for the integration tests. There's no specific reference to it in the code but perhaps a binary installed by the package. It's also possible that the integrations are using [flask-apispec](https://github.com/jmcarp/flask-apispec) for the swagger tests, in which case we can drop the swagger-tools dependency. We'll have to dig in a bit more.

If we move the dependency to the `package.json` we'll have to adjust the `PATH` so the integration tests pick up the right bin path.

**Pull request merged**